src/core/BufferAttribute.d.ts

```diff
@@ -6,6 +6,7 @@ export class BufferAttribute {
 	constructor( array: ArrayLike<number>, itemSize: number, normalized?: boolean ); // array parameter should be TypedArray.
 
 	uuid: string;
```

Comment:
I guess this line is wrong?

Reply:
Yes, `BufferAttribute` has no `uuid`.

Reply:
@mrschofield Would you like to correct this with another PR? 😇

Reply:
@Mugen87 done - https://github.com/mrdoob/three.js/pull/17019